dataplane/saiserver/acl.go

```diff
@@ -292,3 +292,125 @@ func (a *acl) CreateAclEntry(ctx context.Context, req *saipb.CreateAclEntryReque
 
 	return &saipb.CreateAclEntryResponse{Oid: id}, nil
 }
+
+type myMacInfo struct {
+	priority       *uint32
+	portID         *uint64 // wildcard if not specified
+	vlanID         *uint32 // wildcard if not specified
+	macAddress     []byte
+	macAddressMask []byte
+}
+
+// ToEntryDesc returns the EntryDesc.
+func (mi *myMacInfo) ToEntryDesc(m *myMac, withPriority bool) (*fwdpb.EntryDesc, error) {
+	ed := &fwdpb.EntryDesc{
```

Comment:
there is flow entry builder you could use instead https://github.com/openconfig/lemming/blob/main/dataplane/forwarding/fwdconfig/table.go#L167